src/components/DpMultiselect/DpMultiselect.vue

```diff
@@ -26,31 +31,37 @@
       </template>
 
       <template v-slot:option="props">
```

Comment:
Can we also just expose option in the slots? Then, we would not have to adapt all useages.

```suggestion
      <template v-slot:option="{ option }">
```

Reply:
I tried this, but then I couldn't access the remove function and it did not work (`@click="props.remove(props.option)"`.